crates/topos-p2p/src/config.rs

```diff
@@ -32,6 +32,9 @@ pub struct DiscoveryConfig {
     pub provider_publication_interval: Option<Duration>,
     /// Interval at which the node will send bootstrap query to the network
     pub bootstrap_interval: Duration,
+    /// Interval at which the node will send fast bootstrap query to the network
+    /// Mostly used when the node is bootstrapping and failed to connect to boot peers
```

Comment:
```suggestion
    /// Mostly used when the node is bootstrapping and failed to connect to boot peers.
	/// Defaults to 5 sec (['FAST_BOOTSTRAP_INTERVAL`]).
```

(Although I'm not sure if that constant should be `pub`)

Reply:
This `DiscoverConfig` may end up in `topos-config` so I'm ok to let it public